scripts/downsample.sh

```diff
@@ -6,7 +6,12 @@ set -e
 # import logging/versioning
 source /mnt/hds/proj/bioinfo/SCRIPTS/log.bash
 
-VERSION=1.4.1
+#activate production conda env
+shopt -s expand_aliases
+source ${HOME}/.bashrc
+useprod
+
+VERSION=1.4.2
```

Comment:
no need to bump the version yourself. We will bump as soon as we mere to master.

Reply:
Alrighty!

Reply:
How about now? :D